go.mod

```diff
@@ -14,11 +14,11 @@ require (
 	go.uber.org/goleak v1.1.12
 	golang.org/x/sync v0.0.0-20220601150217-0de741cfad7f
 	helm.sh/helm/v3 v3.9.2
-	k8s.io/api v0.24.2
-	k8s.io/apimachinery v0.24.2
+	k8s.io/api v0.25.2
```

Comment:
Contains fix for testing lib https://github.com/kubernetes/kubernetes/pull/110425